### ISSUE TITLE
Create travisci release process

### ISSUE THREAD

**Issue by rspieldenner** [Netflix/feign#205]
Get me out of the loop for releasing this project.

When scripts and keys are in place the process will be just creating a tag, e.g. to release 8.1.0 you would just create a v8.1.0 tag

**Comment by adriancole** [comment]
+100

**Comment by adriancole** [comment]
trying process w/7.4.0

```bash
$ git tag v7.4.0
$ git push --tags
Username for 'https://github.com': adriancole
Password for 'https://adriancole@github.com': 
Total 0 (delta 0), reused 0 (delta 0)
To https://github.com/Netflix/feign.git
 * [new tag]         v7.4.0 -> v7.4.0
```

**Comment by adriancole** [comment]
trying an experiment (last one for today). It appears as though making a commit whose only message is the version number kicked off an rxjava release.

https://travis-ci.org/ReactiveX/RxJava/builds/53487193

Giving that a swing.. I removed the v7.4.0 tag. Then, I added a useless commit to the 7.x branch with message 7.4.0. Let's see!

**Comment by adriancole** [comment]
didn't work. https://travis-ci.org/Netflix/feign/builds/53890802

I'm going to leave this to the pros :)

**Comment by adriancole** [comment]
tried updating to latest nebula (#209)
pushed v7.4.0 tag again and seeing if any luck.
https://travis-ci.org/Netflix/feign/builds/54049931

**Comment by adriancole** [comment]
nope. @rspieldenner over to you when you get a chance. There's no pressing need for a release in the next week anyway.

**Comment by adriancole** [comment]
whoot! works.

I released 7.4.0 by doing the following:
```bash
$ git tag v7.4.0
$ git push --tags
# watch https://travis-ci.org/Netflix/feign/builds
# find https://bintray.com/netflixoss/maven/feign/7.4.0/view
```

**Comment by adriancole** [comment]
and.. documented https://github.com/Netflix/feign/wiki/Releasing-Feign
